src/coreclr/jit/codegeninterface.h

```diff
@@ -173,6 +173,11 @@ class CodeGenInterface
     bool genUseOptimizedWriteBarriers(GenTreeStoreInd* store);
     CorInfoHelpFunc genWriteBarrierHelperForWriteBarrierForm(GCInfo::WriteBarrierForm wbf);
 
+#ifdef DEBUG
+    bool genWriteBarrierUsed;
+    void WriteBarrierExtraSuperPmiQueries();
```

Comment:
Looks like you were going to use a function, but then you didn't?

Reply:
Yep. Removed.